Decred Wallet/Utils/DecredSegue.swift

```diff
@@ -0,0 +1,14 @@
+//
+//  DecredSegue.swift
+//  Decred Wallet
+//
+//  Created by kayeli dennis on 18/06/2019.
+//  Copyright © 2019 Decred. All rights reserved.
+//
+
+import Foundation
+
+enum DecredSegue: String {
```

Comment:
I love the introduction of this file, keep things neat. Seeing that most files/objects used in this project are decred related, it isn't necessary to use a `Decred` prefix or suffix. I'm thinking `Segues` or `AppSegues` (should ideally be singular though) will work. Similar to the `Storyboards` enum in the `Storyboards.swift` file.